man/logind.conf.xml

```diff
@@ -297,6 +297,13 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>InhibitorsMax=</varname></term>
+
+        <listitem><para>Controls the maximum number of concurrent inhibitors to permit. Defaults to 8192
+        (8K).</para></listitem>
+      </varlistentry>
```

Comment:
RLIMIT_NOFILE × RLIMIT_NPROC is higher than 8192 for pretty much any user. So with this change any user is able to saturate the cap on inhibitors and deny other users creation of inhibitors. AFAIK programs treat inhibition taking as non-fatal (e.g. rpm), so this might not be enough to cause other things to fail, but still.